guides/common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc

```diff
@@ -1,12 +1,17 @@
-[id='preparing-server-and-proxies']
-[id='preparing-satellite-server-and-capsule-servers']
-= Prerequisites for Configuring {SmartProxyServer}s for Load Balancing
+[id="Prerequisites_for_Configuring_{smart-proxy-context}_Servers_for_Load_Balancing_{context}"]
+= Prerequisites for Configuring {SmartProxyServerTitle}s for Load Balancing
 
-To configure {SmartProxyServer}s for load balancing, complete the following procedures described in _Installing {SmartProxyServer}_.
+ifdef::orcharhino[]
+You can find a list of requirements for {SmartProxyServer} in xref:sources/installation_and_maintenance/installing_orcharhino_proxy.adoc[_{InstallingSmartProxyDocTitle}_].
```

Comment:
By the way, this is a very interesting use of xref. Is there somewhere more info written on this syntax?

Reply:
I use this throughout the downstream docs. After this PR, I will upstream all currently used attributes for orcharhino & basically replace this with `InstallingSmartProxyDocURL`.

It's basically a relative path; so the attribute minus `docs.theforeman.org/3.5/`.